nautobot/dcim/models/cables.py

```diff
@@ -256,6 +252,8 @@ def clean(self):
             self.length_unit = ""
 
     def save(self, *args, **kwargs):
+        # Avoid Circular Import
+        from nautobot.core.utils import to_meters
```

Comment:
Yuck. We should always be able to import `core` from any other app. Can we at least mark this as a TODO?